traveler-ios-ui/TravelerKitUI/ViewControllers/CatalogItemDetailsViewController.swift

```diff
@@ -7,7 +7,31 @@
 //
 
 import UIKit
+import TravelerKit
 
 class CatalogItemDetailsViewController: UIViewController {
-    
+
+    var itemDetails: CatalogItemDetails?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        switch itemDetails {
+        case let itemDetails as BookingItemDetails:
+            performSegue(withIdentifier: "bookingSegue", sender: itemDetails)
+        default:
+            Log("Unknown item type", data: nil, level: .error)
+            performSegue(withIdentifier: "errorSegue", sender: nil)
+        }
+    }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch (segue.identifier, segue.destination, sender) {
+        case (_, let vc as BookingItemDetailsViewController , let sender as BookingItemDetails):
+            vc.bookingItemDetails = sender
+        default:
+            performSegue(withIdentifier: "errorSegue", sender: nil)
```

Comment:
perform a segue in prepareForSegue??

Log and break, this is already going to the errorSegue

Reply:
Yep, brain fart